### PR TITLE
List card component

### PR DESCRIPTION
**Pull request by mun625**
implement independent card and list component and its container & reducer form.

Please review for the next step.
(Test code will be added right after merge cause size of changed file is getting larger)

### CODE REVIEW

File: src/shared/components/AddCardForm.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    return (
+      <div>
+        <form onSubmit={e => {
+          e.preventDefault();
+          if (!input.value.trim()) {
+            return;
+          }
+          this.props.dispatch(addCard(input.value));
+          input.value = '';
```
Comment (elegant651) [inline src/shared/components/AddCardForm.js:16]: why don't you separate the function from render element ?

File: src/shared/reducers/cards.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import update from 'react-addons-update';
+
+const initialState = {
+  cards: []
+};
+
+const card = (state = initialState, action) => {
+  switch (action.type) {
+  case ADD_CARD:
+    let thisData = {
```
Comment (elegant651) [inline src/shared/reducers/cards.js]: 'let' should be 'const'

File: src/shared/reducers/map.js [hunk 3]
```diff
@@ -9,10 +10,11 @@ const initialState = {
   },
   select: {
     data: {}
-  }
+  },
+  cards: []
 };
 
-export default (state, action) => {
+export default (state = initialState, action) => {
```
Comment (elegant651) [inline src/shared/reducers/map.js]: duplicate setting initialState

File: src/test/browser/components/index.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```
Comment (elegant651) [inline src/test/browser/components/index.js]: discard this file

File: src/shared/components/AddCardForm.js (hunk 1 above)
Comment (ovekyc) [reply src/shared/components/AddCardForm.js:16]: +1

Comment (mun625) [reply src/shared/components/AddCardForm.js:16]: For next step, yuyeong will revise overall detailed add-format and add-reducer function.

**Comment by elegant651** [comment]
LGTM

**Pull request merged**